RFEM6_Adapter/Convert/ToRFEM6/BHoMDataStructure/Loading/Load.cs

```diff
@@ -142,17 +142,21 @@ public static rfModel.surface_load ToRFEM6(this AreaUniformlyDistributedLoad bhA
 
             surface_load_load_direction loadDirecteion;
             Vector orientationVector = bhAreaLoad.Pressure;
+            double magnitude;
             if (orientationVector.X != 0)
             {
-                loadDirecteion = bhAreaLoad.Projected ? surface_load_load_direction.LOAD_DIRECTION_GLOBAL_X_OR_USER_DEFINED_U_PROJECTED : surface_load_load_direction.LOAD_DIRECTION_LOCAL_X;
```

Comment:
Would fix the names to loadDirection instead of typo loadDirecteion ;)

Reply:
I am assuming that you are referring to the push of free_line loads?